.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  all:
+    runs-on: windows-latest

```

Comment:
Errors about a Sqlite binary on linux. Assumedly fixed in newer versions, but will look after getting things going.

Reply:
It's also insanely slow to do the `dotnet build` on windows runners for whatever reason (2m26s vs 19s), so would be nice to switch